x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/enrich/action/EnrichStatsAction.java

```diff
@@ -81,26 +79,18 @@ public void writeTo(StreamOutput out) throws IOException {
         public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
             builder.startObject();
             builder.startArray("executing_policies");
-            executingPolicies.stream().sorted(Comparator.comparing(ExecutingPolicy::getName)).forEachOrdered(policy -> {
-                try {
-                    builder.startObject();
-                    policy.toXContent(builder, params);
-                    builder.endObject();
-                } catch (IOException e) {
-                    throw new UncheckedIOException(e);
-                }
-            });
+            for (ExecutingPolicy policy : executingPolicies) {
```

Comment:
moved sorting to `TransportEnrichStatsAction` to make testing easier in `StatsResponseTests`.

Reply:
prolly a better place for it anyway :)